core/src/test/java/org/elasticsearch/plugins/PluginManagerUnitTests.java

```diff
@@ -63,8 +62,8 @@ public void testThatConfigDirectoryCanBeOutsideOfElasticsearchHomeDirectory() th
         Environment environment = new Environment(settings);
 
         PluginManager.PluginHandle pluginHandle = new PluginManager.PluginHandle(pluginName, "version", "user");
-        String configDirPath = Files.simplifyPath(pluginHandle.configDir(environment).normalize().toString());
-        String expectedDirPath = Files.simplifyPath(genericConfigFolder.resolve(pluginName).normalize().toString());
+        String configDirPath = pluginHandle.configDir(environment).normalize().toString();
+        String expectedDirPath = genericConfigFolder.resolve(pluginName).normalize().toString();
```

Comment:
Hmm, I don't think we still need simplifyPath if we do normalize() here, what do you think?